packages/react/src/components/logo/Logo.module.css

```diff
@@ -0,0 +1,17 @@
+.logo {
+  max-width: 100%;
+  height: 100%;
+  max-height: 100%;
```

Comment:
Why both `height` and `max-height` as `100%`?

Reply:
I guess the thought was that the logo would never overflow the container. E.g. if you set the size to large and the height of the container would be less than the 3rem height of the large logo, then without the max-height it would overflow the container.